src/services/bindService.js

```diff
@@ -188,19 +188,12 @@ angular.module("openshiftCommonServices")
     };
 
     var sortServiceInstances = function(serviceInstances, serviceClasses) {
-      if (!serviceInstances && !serviceClasses) {
-        return null;
-      }
+      var getServiceClassDisplayName = function(serviceInstance) {
+        var serviceClassName = _.get(serviceInstance, 'spec.clusterServiceClassRef.name');
+        return _.get(serviceClasses, [serviceClassName, 'spec', 'externalMetadata', 'displayName']) || serviceInstance.spec.clusterServiceClassExternalName;
```

Comment:
just checking, external name is required right?

Reply:
I'm not sure it is, but it should be OK if it's not set?

<img width="530" alt="screen shot 2017-10-26 at 12 47 01 pm" src="https://user-images.githubusercontent.com/1167259/32065815-d46e5d0c-ba4b-11e7-98e9-89d617ede055.png">